src/api/procedures/__tests__/removeSecondaryAccounts.ts

```diff
@@ -85,36 +101,35 @@ describe('removeSecondaryAccounts procedure', () => {
 
   test('should throw an error if attempting to remove the primary Account', () => {
     const proc = procedureMockUtils.getInstance<RemoveSecondaryAccountsParams, void>(mockContext);
-    const signer = entityMockUtils.getAccountInstance({ address: 'primaryAccount' });
-
-    signerToSignerValueStub
-      .withArgs(signer)
-      .returns({ type: SignerType.Account, value: signer.address });
 
     return expect(
       prepareRemoveSecondaryAccounts.call(proc, {
         ...args,
-        signers: [signer],
+        accounts: [account],
+        identity: entityMockUtils.getIdentityInstance({
+          getPrimaryAccount: primaryAccount,
+          getSecondaryAccounts: [],
+        }),
       })
     ).rejects.toThrow('You cannot remove the primary Account');
   });
 
   test('should throw an error if at least one of the secondary Accounts to remove is not present in the secondary Accounts list', () => {
-    const { signers } = args;
-    const signerValue = { type: SignerType.Account, value: (signers[0] as Account).address };
+    const { accounts } = args;
+    const signerValue = { type: SignerType.Account, value: accounts[0].address };
 
-    signerToSignerValueStub.withArgs(signers[0]).returns(signerValue);
+    signerToSignerValueStub.withArgs(accounts[0]).returns(signerValue);
 
     const proc = procedureMockUtils.getInstance<RemoveSecondaryAccountsParams, void>(mockContext);
 
     return expect(
       prepareRemoveSecondaryAccounts.call(proc, {
         ...args,
         identity: entityMockUtils.getIdentityInstance({
-          getPrimaryAccount: entityMockUtils.getAccountInstance({ address: 'primaryAccount' }),
+          getPrimaryAccount: primaryAccount,
           getSecondaryAccounts: [],
         }),
```

Comment:
This shouldn't be necessary either. This procedure's args don't have an `identity` property